source/_integrations/bzutech.markdown

```diff
@@ -0,0 +1,22 @@
+---
+title: Bzu Tech
+description: Instructions on how to setup BzuTech devices within Home Assistant.
+ha_category:
+  - Sensors
+ha_release: 2024.1
+ha_iot_class: Cloud Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@enioo22'
+ha_domain: bzutech
+ha_quality_scale: platinum
+ha_platforms:
+  - sensor
+ha_integration_type: integration
+---
+
+The **Bzu Tech** {% term integration %} allows you to integrate your [BzuCloud](https://cloud.bzutech.com.br) sensors and devices* in Home Assistant, generating a **Sensor {% term Entity %}** that can be used in automations, dashboards and more.
+
+{% include integrations/config_flow.md %}
+
+*To use this integration you need a BzuCloud account with registered devices.
```

Comment:
```suggestion

## Prerequisites

To use this integration, you need a BzuCloud account with registered devices.

{% include integrations/config_flow.md %}

```

Reply:
is there anything you can say about the type of sensors? Humidity, temperature, ...?

Reply:
> is there anything you can say about the type of sensors? Humidity, temperature, ...?

the integration is compatible with every type of bzu sensors: humidity, temperature, luminosity, co, co2, volatile organic compounds, voltage and current, do you think i should add it to the documentation??